src/vs/workbench/contrib/debug/browser/debug.contribution.ts

```diff
@@ -489,6 +489,16 @@ function registerConfiguration(): void {
 				type: 'boolean',
 				description: nls.localize({ comment: ['This is the description for a setting'], key: 'showInlineBreakpointCandidates' }, "Controls whether inline breakpoints candidate decorations should be shown in the editor while debugging."),
 				default: true
+			},
+			'debug.saveBeforeStart': {
+				description: nls.localize('debug.saveBeforeStart', "Save all dirty editors before starting a debug session."),
```

Comment:
Please update description to be better
`Controls what editors to save before starting a debug session.`